lib/spack/spack/test/config.py

```diff
@@ -436,20 +437,35 @@ def test_merge_with_defaults(mock_low_high_config, write_config_file):
     assert cfg['baz']['version'] == ['c']
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
 def test_substitute_user(mock_low_high_config):
     user = getpass.getuser()
     assert '/foo/bar/' + user + '/baz' == spack_path.canonicalize_path(
         '/foo/bar/$user/baz'
     )
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
+def test_substitute_user_config(mock_low_high_config):
```

Comment:
This test content looks the same as the next test to me, and both are skipped on Windows, so I'm not sure why it was added in this PR.

Reply:
It's passing now.